CMakeLists.txt

```diff
@@ -29,7 +29,11 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 include(GNUInstallDirs)
 
-cmake_policy(SET CMP0167 OLD)  # libboost-headers packaged by conda does not come with BoostConfig.cmake
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.30.0") 
+#  libboost-headers packaged by conda does not come with BoostConfig.cmake
+cmake_policy(SET CMP0167 OLD)  
+endif()
+
 find_package(Boost REQUIRED)
```

Comment:
```suggestion
cmake_policy(PUSH)
if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.30.0") 
  #  libboost-headers packaged by conda does not come with BoostConfig.cmake
  cmake_policy(SET CMP0167 OLD)  
endif()
find_package(Boost REQUIRED)
cmake_policy(POP)
```

Reply:
I don't think we need such a complicated stuff. Setting `CMP0167` permanently (for this project) would not hurt.